apps/src/templates/types.js

```diff
@@ -25,3 +26,9 @@ export const rubricShape = PropTypes.shape({
   performanceLevel3: PropTypes.string,
   performanceLevel4: PropTypes.string
 });
+
+export const ReviewStates = makeEnum(
```

Comment:
(minor) Wondering if we should we rename this to something with teacher feedback (since it's now in a top-level file and it's not clear that it's related to teacher feedback).

Reply:
I actually plan on adding a directory to instructions that will hold all teacher feedback components. I'll move this to that place when it exists. This is the next task on my list after this PR is merged.